### PR TITLE
Update README with conda installation

### PR DESCRIPTION
**Pull request by maresb**
For after https://github.com/conda-forge/staged-recipes/pull/21054 is merged

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,13 +7,14 @@ PyMC-BART extends [PyMC](https://github.com/pymc-devs/pymc) probabilistic progra
 
 ## Installation
 
-PyMC-BART requires a working Python interpreter (3.8+). We recommend installing Python and key numerical libraries using the [Anaconda Distribution](https://www.anaconda.com/products/individual#Downloads), which has one-click installers available on all major platforms.
+PyMC-BART is available on Conda-Forge. To set up a suitable Conda environment, run
```
Comment (aloctavodia) [inline README.md:10]: Instructions for "a pure pip" option should still be available

Comment (maresb) [reply README.md:10]: Done. Any further suggestions?

**Comment by aloctavodia** [comment]
Thanks  @maresb!

**Comment by maresb** [comment]
To be completely explicit, these conda-forge instructions may be invalid for some days until the recipe is reviewed and published. If this is problematic, then this should be reverted.

**Pull request merged**